### PR TITLE
Add submit button for repl on mobile

### PR DESCRIPTION
**Pull request by gregorybchris**
Enter works for submitting, but it's not always obvious on mobile

Screenshot 2023-12-29 at 5 52 28 PM

### CODE REVIEW

File: repl.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
 Output:
 </div>
 <div>
-<code><span id="prompt-string">>>></span></code><input id="input" type="text" />
+<input id="input" type="text" /><input id="input-button" type="submit" value="Submit" onclick=submitInput />
```
Comment (tekknolagi) [inline repl.html]: can you register the onClick in the JS please?

File: repl.html [hunk 2]
```diff
@@ -120,38 +120,45 @@
...
+    }
+}
+
 loadFromLocalStorage();
 input.focus();
 async function inputHandler(e) {
     // TODO(max): Make up/down arrow keys navigate history
     if (e.key === "Enter") {
-        if (e.shiftKey && input.tagName === "INPUT") {
+        if (input.tagName === "INPUT" && e.shiftKey) {
```
Comment (tekknolagi) [inline repl.html]: Can you put this back how it was

File: repl.html [hunk 3]
```diff
@@ -120,38 +120,45 @@
...
             // Shift-Enter expands the input to a textarea and does not submit
             // input.
             expandInput();
             return;
         }
-        if (!e.shiftKey && input.tagName === "TEXTAREA") {
```
Comment (tekknolagi) [inline repl.html:134]: Why remove this?

File: repl.html [hunk 4]
```diff
@@ -120,38 +120,45 @@
...
-            // Normal Enter in a textarea should not submit input.
-            return;
-        }
-        const response = await sendRequest(document.env, input.value);
```
Comment (tekknolagi) [inline repl.html:138]: Any particular reason to extract into a function?

Comment (gregorybchris) [reply repl.html:138]: We want to use it in both on enter and on submit press

File: repl.html (hunk 2 above)
Comment (gregorybchris) [reply repl.html]: I did this to match the structure of the textarea conditional below. Can put it back if you feel strongly.

File: repl.html (hunk 3 above)
Comment (gregorybchris) [reply repl.html:134]: The logic still exists below

File: repl.html (hunk 1 above)
Comment (gregorybchris) [reply repl.html]: Sure but what does that mean?

File: repl.html (hunk 2 above)
Comment (tekknolagi) [reply repl.html]: Eh, not necessary for the diff. I tend to leave as-is as much as possible (leaving stylistic changes to their own commit(s))

Comment (tekknolagi) [reply repl.html]: Makes reverting easy

Comment (tekknolagi) [reply repl.html]: I don't care a lot for this in particular either way

File: repl.html (hunk 1 above)
Comment (tekknolagi) [reply repl.html]: `#input-button.addEventListener` etc like the others

File: repl.html (hunk 3 above)
Comment (tekknolagi) [reply repl.html:134]: oice

File: repl.html (hunk 4 above)
Comment (tekknolagi) [reply repl.html:138]: oic

**Review by tekknolagi — APPROVED** [review]
lgtm minus small nits

File: repl.html (hunk 1 above)
Comment (gregorybchris) [reply repl.html]: Done!

File: repl.html (hunk 2 above)
Comment (gregorybchris) [reply repl.html]: Alright, as you wish

**Pull request merged**